spec/rubocop/cop/style/safe_navigation_spec.rb

```diff
@@ -44,16 +44,26 @@
       expect_no_offenses('foo && !foo.bar.baz?')
     end
 
-    it 'allows method call that are used in a comparison safe guarded by ' \
+    it 'allows method call that is used in a comparison safe guarded by ' \
       'an object check' do
       expect_no_offenses('foo.bar > 2 if foo')
     end
 
-    it 'allows an object check before a method call that are used in ' \
+    it 'allows method call that is used in a non standard comparison ' \
+      'safe guarded by an object check' do
+      expect_no_offenses('foo.bar !~ /baz/ if foo')
```

Comment:
I don't see tests for `=~` and the spaceship operator. And wouldn't not necessarily call them non-standard comparisons. :-) I'd just go with `regexp matching` and `spaceship operator`.